README.md

```diff
@@ -37,10 +37,6 @@ has the following endpoints:
 * `/debug/pprof` is a
   [net/http/pprof](https://golang.org/pkg/net/http/pprof/) endpoint.
 
-In the configuration file, the `index_extra_html` may contain HTML
-that will be inserted at the top of the file, after the server start
-time and current host:port.
-
 ## Metrics
 
 The following metrics are collected by Prometheus:
```

Comment:
We should probably update this part of the readme since it's now outdated. And other stuff in the readme. I'm happy to do that in a separate pr if you'd like.

Reply:
I'm kind of inclined to point the metrics doc section at the godoc page for the metrics package; maintaining the metrics text in two spots is annoying.

I'd say the same for the configs, but that needs to be refactored to support that.

Either way, if you don't mind- a followup PR would be appreciated.  Next change in the PR pipeline from me is lint/vet fixes, and then back to the actual bugremoval work.

Reply:
Sure thing, I'll make a readme PR Monday morning.